rcswitch/user_doc.rst

```diff
@@ -0,0 +1,134 @@
+.. index:: Plugins; rcswitch (Brennenstuhl RCS 1000 N)
+.. index:: rcswitch
+
+================
+Plugins/rcswitch
+================
+
+
+Anforderungen
```

Comment:
Die Haupt Abschnitte der Dokumentation sollten '=' zur Kennzeichnung nutzen, also

```
Anforderungen
=============
```

Die nächste Unterebene sollte dann mit '-' Zeichen gekennzeichnet sein (statt mit '~' Zeichen).

Reply:
erl. in PR für `rcs1000n`